lighthouse-core/formatters/partials/critical-request-chains.html

```diff
@@ -89,7 +95,8 @@
         <span class="cnc-node__tree-hostname">({{ this.hostname }})</span>
       {{/parseURL}}
       {{#unless hasChildren}}
-        - <span class="cnc-node__chain-duration">{{chainDuration startTime this.node.request.endTime }}ms, {{formatTransferSize this.transferSize}}KB</span>
+        - <span class="cnc-node__chain-duration">{{chainDuration startTime this.node.request.endTime }}ms</span>,
+          <span class="cnc-node__chain-duration">{{formatTransferSize this.transferSize}}KB</span>
```

Comment:
nit: not really chain-duration, why was it split into two spans? or maybe use a new `chain-size` modified or something?